src/intro/hello-world.md

```diff
@@ -95,6 +95,13 @@ The main crate of gdext is called `godot`. At this point, it is still hosted on
 To fetch the latest changes, you can regularly run a `cargo update` (possibly breaking). Keep your `Cargo.lock` file under version control,
 so that it's easy to revert updates.
 
+```admonish note
+It may be a advisable to put a [.gdignore][gd-ignore] file at the top-level of your rust project directory or within your `/target` dir.
+This is because for some Windows build targets, `cargo build` produces an object file with the `.obj` extension rather than the typical `.o`
```

Comment:
I believe this message gives the slight impression that this is only recommended for Windows targets, when, in reality, this should be recommended for all platforms. Also, it should probably be made more explicit that this is only necessary if your Rust code is inside the Godot project folder (containing `project.godot`); having separate `godot/` and `rust/` folders would also solve the issue.

Reply:
> having separate `godot/` and `rust/` folders would also solve the issue.

Btw, I just added that as the recommended directory structure in the Hello World tutorial 🙂 
https://godot-rust.github.io/book/intro/hello-world.html#directory-setup

Reply:
Nice! Sounds like we could benefit from a direct link to that section then. Something like this:

"If you do not follow the [recommended directory structure setup for `gdext` projects](https://godot-rust.github.io/book/intro/hello-world.html#directory-setup) of having separate `godot/` and `rust/` folders, and instead place your Rust code root folder directly inside the Godot project folder (the one containing `project.godot`), then consider adding a `.gdignore` file at the root folder of your Rust code to avoid having the Godot editor accidentally and inappropriately attempt to import files from your Rust folder, such as `.obj` files which might be generated by `cargo build` and placed in your `target` subfolder, or other possibly conflicting files which the Rust compiler might generate."